recipes/pcraster/meta.yaml

```diff
@@ -0,0 +1,103 @@
+{% set version = "4.3.0" %}
+
+package:
+  name: pcraster
+  version: {{ version }}
+
+source:
+  url: http://pcraster.geo.uu.nl/pcraster/packages/src/pcraster-{{ version }}.tar.bz2
+  sha256: a4be969033d7d08571eca7a5ca27412f3d1a8e1d1ea20d41f6c82e74442c2352
+
+
+build:
+  number: 0
+  skip: True                                # [py2k]
+
+
+  missing_dso_whitelist:
+    - C:/Windows/System32/MSVCP140.dll      # [win]
+    - C:/Windows/System32/VCRUNTIME140.dll  # [win]
+    - C:/Windows/System32/imagehlp.dll      # [win]
+    - C:/Windows/System32/GLU32.dll         # [win]
+    - C:/Windows/System32/OPENGL32.dll      # [win]
+    - C:/Windows/System32/dwmapi.dll        # [win]
+    - C:/Windows/System32/WTSAPI32.dll      # [win]
+
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}         # [linux]
+    - {{ cdt('mesa-dri-drivers') }}         # [linux]
+    - {{ cdt('libselinux') }}               # [linux]
+    - {{ cdt('libxdamage') }}               # [linux]
+    - {{ cdt('libxxf86vm') }}               # [linux]
+    - cmake >=3.15
+    - ninja
+    - make                                  # [win]
+
+  host:
+    - xorg-libxfixes                        # [linux]
+    - boost-cpp
+    - xerces-c
+    - qt
+    - ncurses                               # [not win]
+    - gdal
+    - libgdal
+    - python
+    - numpy 1.14.*
+    - vs2015_runtime                        # [win]
+    - libglu                                # [linux]
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - boost-cpp
+    - libgdal
+    - ncurses                               # [not win]
+    - vs2015_runtime                        # [win]
+    - libglu                                # [linux]
+    - zlib                                  # [win]
+    - libpng                                # [win]
+
+
+
+test:
+    imports:
+       - pcraster
+    commands:
+        - asc2map
+        - col2map
+        - legend
+        - map2asc
+        - map2col
+        - mapattr
+        - oldcalc
+        - pcrcalc
+        - resample
+        - table
```

Comment:
Do these have help or version commands you can run?

Reply:
The applications don't have a --version or similar (due to historical reasons). Without any arguments given, the PCRaster applications show version number and usage information.

Reply:
OK!